pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -323,7 +333,12 @@ func (r ReplicaSetReconciler) buildAutomationConfigConfigMap(mdb mdbv1.MongoDB)
 	if err != nil {
 		return corev1.ConfigMap{}, fmt.Errorf("error reading version manifest from disk: %+v", err)
 	}
-	ac := buildAutomationConfig(mdb, manifest.BuildsForVersion(mdb.Spec.Version))
+	currentCm := &corev1.ConfigMap{}
+	currentAc := automationconfig.AutomationConfig{}
```

Comment:
[nit] there's a stylistic inconsistency here

at first we get a reference, then pass the reference directly, but then we create the struct, and pass it by reference

I would prefer

```golang
currentCm := corev1.ConfigMap{}
currentAc := automationconfig.AutomationConfig{}

// use &currentCm
// use &currentAc
```